### PR TITLE
OSSMDOC-323: Update oc login

### PR DESCRIPTION
**Pull request by rh-tokeefe**
Version(s): 4.6-4.12

Issue: https://issues.redhat.com/browse/OSSMDOC-323

Link to docs preview: http://file.bos.redhat.com/tokeefe/OSSMDOC-323/service_mesh/v2x/ossm-about.html

Additional information:

### CODE REVIEW

**Comment by prachiyadav** [comment]
Verified by QE

File: modules/jaeger-deploy-production-es.adoc [hunk 1]
```diff
@@ -90,7 +90,7 @@ Follow this procedure to create an instance of Jaeger from the command line.
 +
 [source,terminal]
 ----
-$ oc login https:/<HOSTNAME>:8443
+$ oc login --username=<NAMEOFUSER> https:/<HOSTNAME>:8443
```
Comment (jwendell) [inline modules/jaeger-deploy-production-es.adoc:93]: missing one `/` in `https://`

File: modules/jaeger-removing-instance-cli.adoc [hunk 2]
```diff
@@ -11,7 +11,7 @@ This module included in the following assemblies:
 +
 [source,terminal]
 ----
-$ oc login
+$ oc login --username=<NAMEOFUSER>
```
Comment (jwendell) [inline modules/jaeger-removing-instance-cli.adoc:14]: should this contain the `https://<HOSTNAME>:8443` part too?

File: modules/ossm-control-plane-cli.adoc [hunk 3]
```diff
@@ -19,7 +19,7 @@ You can deploy a basic `ServiceMeshControlPlane` from the command line.
 +
 [source,terminal]
 ----
-$ oc login https://<HOSTNAME>:6443
+$ oc login --username=<NAMEOFUSER> https://<HOSTNAME>:6443
```
Comment (jwendell) [inline modules/ossm-control-plane-cli.adoc:22]: Some places mention port `8443` and some `6443`. Which one is correct? Both?

File: modules/jaeger-removing-instance-cli.adoc (hunk 2 above)
Comment (rh-tokeefe) [reply modules/jaeger-removing-instance-cli.adoc]: This file was removed as part of work Julie performed for another story, and is no longer part of this effort.

File: modules/ossm-control-plane-cli.adoc (hunk 3 above)
Comment (rh-tokeefe) [reply modules/ossm-control-plane-cli.adoc:22]: Looks like both are correct from what I can gather.

**Pull request merged**